### ISSUE TITLE
smpl pose illustration when get embedding

### ISSUE THREAD

**Issue by chrenx** [Sirui-Xu/InterDiff#10]
Hello Sirui,
I was confused by the function _get_embeddings() in the file diffusion_smpl.py that you concatenate the body pose in both past and the future.

 body_pose = torch.cat([frame['smplfit_params']['pose'][:, :66].unsqueeze(0) for frame in data['frames']], dim=0).float().to(device)
 
Why did you choose 66 here? I print out the original pose shape from behave dataset which was TxBx156. Could you illustrate a little bit here?

Best,
chrenx

**Comment by chrenx** [comment]
I actually found out that 66 may be due to MANO which replaced smpl hand with mano model. 3 + 21 x 3 if we don't count the hand, this would be 66. Is this right?

**Comment by Sirui-Xu** [comment]
Hello @chrenx,

Yes, you are correct. The number 66 is for the body pose other than hands. The reasons are:

The emphasis in this work is on the body parts other than the hands, which, despite their importance, are represented by fewer parameters in SMPL. And we find training without hand poses is helpful.
The BEHAVE dataset, at the time of our submission, did not offer detailed data on hand poses. I'm not sure if this has been released recently. So, our method just replicates the final hand pose from the past sequence for the generation.

Best regards,
Sirui
